tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
> Given your penchant for small diffs on the autogenerated docs, I'm a bit surprised you'd want to go this route

Eh? Removing the `static` keyword is a single-line diff; and I'm not worried about the code size of the *unit test* binaries.